### ISSUE TITLE
hyperctl images panic

### ISSUE THREAD

**Issue by ZYNCMA** [hyperhq/hyperd#430]
If I pull an image using the command `hyperctl pull 127.0.0.1:5000/library/ubuntu:latest`, that will cause `hyperctl images` to panic in `hyperhq/hyperd/client/images.go`.
This is because in `images.go`, it uses `:` to split items in `imagesList`, but doesn't handle the case I described above. Fields become `[ip, port/repo, tag, image_id, date, size]` instead of `[repo, tag, image_id, date, size]`.
```
for _, item := range imagesList {
    fields := strings.Split(item, ":")
    date, _ := strconv.ParseUint(fields[3], 0, 64)
    fmt.Fprintf(w, "%s\t%s\t%s\t%s\t%s\n", fields[0], fields[1], fields[2][:12], time.Unix(int64(date), 0).Format("2006-01-02 15:04:05"), getImageSizeString(fields[4]))
}
```
I think a new separator is required.

**Comment by ZYNCMA** [comment]
Maybe something like `rsplitN` is better

**Comment by xiaods** [comment]
PR is welcome

**Comment by ZYNCMA** [comment]
Close this issue as it has been fixed in v0.7.0
